websites/D/DualFM/presence.ts

```diff
@@ -31,22 +31,16 @@ presence.on("UpdateData", async () => {
     largeImageKey: "logo",
     smallImageKey: "dualfm-play-v2"
   };
-
-  const toggleelaspe = await presence.getSetting("toggleelapse");
   const changedetails = await presence.getSetting("changedetails");
   const changestate = await presence.getSetting("changestate");
   const changesmalltext = await presence.getSetting("changesmalltext");
```

Comment:
instead of removing it, give it a default false:
const toggleelaspe = await presence.getSetting("toggleelapse") || true;

Reply:
do that for all of them btw

Reply:
I didn't even think about adding that, you're a genius haha. Thank you :)